docs/core/tutorials/libraries.md

```diff
@@ -66,19 +66,19 @@ Keep in mind that some of the .NET Framework versions used here are no longer in
 
 If you want to reach the maximum number of developers and projects, use the .NET Framework 4.0 as your baseline target. To target the .NET Framework, you will need to begin by using the correct Target Framework Moniker (TFM) that corresponds to the .NET Framework version you wish to support.
 
-```
-.NET Framework 2.0   --> net20
-.NET Framework 3.0   --> net30
-.NET Framework 3.5   --> net35
-.NET Framework 4.0   --> net40
-.NET Framework 4.5   --> net45
-.NET Framework 4.5.1 --> net451
-.NET Framework 4.5.2 --> net452
-.NET Framework 4.6   --> net46
-.NET Framework 4.6.1 --> net461
-.NET Framework 4.6.2 --> net462
-.NET Framework 4.7   --> net47
-```
+| .NET Framework version | TFM    |
+| ---------------------- | ------ |
+| .NET Framework 2.0     | net20  |
```

Comment:
Great change to use table here! Can you also backtick the TFM values as shown in the following suggestion?

```suggestion
| .NET Framework 2.0     | `net20`  |
```